src/pages/workspace/categories/WorkspaceCategoriesSettingsPage.tsx

```diff
@@ -45,20 +73,33 @@ function WorkspaceCategoriesSettingsPage({policy, route}: WorkspaceCategoriesSet
                 testID={WorkspaceCategoriesSettingsPage.displayName}
             >
                 <HeaderWithBackButton title={translate('common.settings')} />
-                <View style={styles.flexGrow1}>
-                    <ToggleSettingOptionRow
-                        title={translate('workspace.categories.requiresCategory')}
-                        subtitle={toggleSubtitle}
-                        switchAccessibilityLabel={translate('workspace.categories.requiresCategory')}
-                        isActive={policy?.requiresCategory ?? false}
-                        onToggle={updateWorkspaceRequiresCategory}
-                        pendingAction={policy?.pendingFields?.requiresCategory}
-                        disabled={!policy?.areCategoriesEnabled || !hasEnabledOptions || isConnectedToAccounting}
-                        wrapperStyle={[styles.mt2, styles.mh4]}
-                        errors={policy?.errorFields?.requiresCategory ?? undefined}
-                        onCloseError={() => Policy.clearPolicyErrorField(policy?.id ?? '-1', 'requiresCategory')}
-                        shouldPlaceSubtitleBelowSwitch
-                    />
+                <ToggleSettingOptionRow
+                    title={translate('workspace.categories.requiresCategory')}
+                    subtitle={toggleSubtitle}
+                    switchAccessibilityLabel={translate('workspace.categories.requiresCategory')}
+                    isActive={policy?.requiresCategory ?? false}
+                    onToggle={updateWorkspaceRequiresCategory}
+                    pendingAction={policy?.pendingFields?.requiresCategory}
+                    disabled={!policy?.areCategoriesEnabled || !hasEnabledOptions || isConnectedToAccounting}
+                    wrapperStyle={[styles.pv2, styles.mh5]}
+                    errors={policy?.errorFields?.requiresCategory ?? undefined}
+                    onCloseError={() => Policy.clearPolicyErrorField(policy?.id ?? '-1', 'requiresCategory')}
+                    shouldPlaceSubtitleBelowSwitch
+                />
+                <View style={[styles.containerWithSpaceBetween]}>
+                    {!!currentPolicy && sections?.length > 0 && (
+                        <SelectionList
+                            headerContent={
+                                <View style={[styles.mh5, styles.mt2]}>
+                                    <Text style={[styles.headerText]}>{translate('workspace.categories.defaultSpendCategories')}</Text>
+                                    <Text style={[styles.mt1, styles.lh20]}>{translate('workspace.categories.spendCategoriesDescription')}</Text>
+                                </View>
+                            }
+                            sections={sections}
+                            ListItem={CategorySelectorListItem}
+                            onSelectRow={() => {}}
```

Comment:
you've used it to disable it, or is it a leftover?

Reply:
I've used it to disable it. I don't really have a need for `onSelectRow` here, but I believe it's not important enough to change the types of `SelectionList` WDYT?

Reply:
either keep it or make it optional, I'd like to avoid regression, but it's a weird way to disable interactions

Reply:
I've already seen it in a couple of places around the app, we'll see what the C+ team has to say 😄

Reply:
I find myself agreeing with @BrtqKr.

Though, maybe better to accept for now and tackle in a separate issue all these cases.

![2024-09-03_11-01-01](https://github.com/user-attachments/assets/5a62ccc2-a80b-4d50-b355-022a1b4f1839)